StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentOptionsViewController.swift

```diff
@@ -92,9 +92,9 @@ class SavedPaymentOptionsViewController: UIViewController {
             return false
         case 1:
             // If there's exactly one PM, customer can only edit if configuration allows removal or if that single PM allows for the card brand choice to be updated.
-            return paymentSheetConfiguration.paymentMethodRemove && (configuration.allowsRemovalOfLastSavedPaymentMethod || viewModels.contains(where: {
+            return (paymentSheetConfiguration.paymentMethodRemove && configuration.allowsRemovalOfLastSavedPaymentMethod) || viewModels.contains(where: {
```

Comment:
I feel like ideally we unit test `canEditPaymentMethods` instead of testing via UI test. That way we can test that various combos of paymentMethodRemove, allowsRemovalOfLastSavedPaymentMethod, and PMs produce the right value. Feel free to ticket.

Reply:
Adding a run ticket!